README.md

```diff
@@ -94,6 +94,8 @@ You also have to provide the following environment variables from an account tha
 - `RESINTEST_REGISTER_PASSWORD`: The password of the account to register.
 - `RESINTEST_REGISTER_USERNAME`: The username of the account to register.
```

Comment:
@thgreasi, sorry, I have no idea what that means. Can you explain some more? What exactly would this look like, and what would it give us?